src/main/java/edu/cornell/kfs/pdp/service/impl/CuPendingTransactionServiceImpl.java

```diff
@@ -959,11 +954,16 @@ protected boolean generateEntriesPaymentRequest(PaymentRequestDocument preq, Lis
                 }
 
             }
-
-          
+            
+            if (CollectionUtils.isNotEmpty(preq.getItems())) {
+                LOG.info("generateEntriesPaymentRequest, cancel previous revisions of preq document: " + preq.getDocumentNumber());
+                purapAccountRevisionService.cancelPaymentRequestAccountRevisions(preq.getItems(), preq.getPostingYearFromPendingGLEntries(), 
+                        preq.getPostingPeriodCodeFromPendingGLEntries());
+            } else {
+                LOG.info("generateEntriesPaymentRequest, unable to cancel previous revisions becuase the items are empty for document: " + preq.getDocumentNumber());
```

Comment:
Minor typo in label; should be changed from "...becuase..." to "...because..." instead.